content/item-hash/index.md

```diff
@@ -0,0 +1,213 @@
+---
+rfc:
+start_date: 2018-08-03
+pr:
+status: draft
+---
+
+# Item hash with redaction
+
+## Summary
+
+This RFC proposes an algorithm to hash items such that data can be redacted
```

Comment:
I guess this should say data can be partially redacted, since we could redact now if we wanted to redact the whole item.